keyboards/littlealby/mute/config.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2019 albybarber
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#pragma once
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x1235 // "AB"
+#define PRODUCT_ID      0x5555 // "MUTE"
```

Comment:
```suggestion
#define VENDOR_ID       0x4142 // "AB"
#define PRODUCT_ID      0x4D55 // "MUTE"
```
Per [this page](https://yanfali.github.io/qmk_usb_usage/) it looks like this VID is currently free.